### PR TITLE
Documentation Review

### PR DESCRIPTION
**Pull request by shmo1218**
### Description
* made doc update
* fixed some md issues

### Solution
* 

### How to apply
Provide steps how to apply on top previous Kubemarine version to execute on existing clusters
* [Install task/s](documentation/Installation.md#installation-tasks-description)
* Manual step 
* [Upgrade procedure](documentation/Maintenance.md#upgrade-procedure)

### Test Cases
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**TestCase 1**

Test Configuration:

- Hardware: 
- OS: 
- Inventory: 

Steps:

1. 

Results:

| Before | After |
| ------ | ------ |
|  |  |

### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [ ] There is no merge conflicts

#### Unit tests
Indicate new or changed unit tests and what they do, if any.

### CODE REVIEW

**Comment by shmo1218** [comment]
Hi, @koryaga,

I have reviewed and edited the doc files with reference to the following pull requests:

https://github.com/Netcracker/KubeMarine/pull/405
https://github.com/Netcracker/KubeMarine/pull/408
https://github.com/Netcracker/KubeMarine/pull/411

Please take a look.

Thanks!

File: documentation/Installation.md [hunk 1]
```diff
@@ -4661,10 +4661,10 @@ Specify the following parameters:
 
 The `chart_path` parameter specifies the absolute path on local host to the Helm chart. The URL link to chart archive is also supported.
 
-The `values_file` parameter specifies the absolute path on local host to the file with YAML formatted values for the chart that override values from `values.yaml` file from the provided chart.
+The `values_file` parameter specifies the absolute path on the local host to the file with YAML formatted values for the chart that overrides values from the `values.yaml` file from the provided chart.
```
Comment (ilia1243) [inline documentation/Installation.md:4664]: values ... override ... values, so probably it was initially correct? Please also update kubemarine\resources\schemas\definitions\plugins\installation\helm.json correspondingly.

Comment (shmo1218) [reply documentation/Installation.md:4664]: Oh, that's right. I will make it "override" like initially. I assumed it as parameter...overrides...values. 
Will also review the json file and inform you.

Thanks!

Comment (shmo1218) [reply documentation/Installation.md:4664]: Hi, @ilia1243,

The helm.json file is reviewed.

**Review by koryaga — APPROVED** [review]

File: documentation/Troubleshooting.md [hunk 2]
```diff
@@ -23,13 +23,12 @@ This section provides troubleshooting information for KubeMarine and Kubernetes
...
-  - [No Pod-to-Pod Traffic for Some Nodes](#no-pod-to-pod-traffic-for-some-nodes)
   - [Long Pulling of Images](#long-pulling-of-images)
-  - [No Pod-to-Pod Traffic for Some Nodes with more than one network interface](#no-pod-to-pod-traffic-for-some-nodes-with-more-than-one-network-interface)
-  - [No Pod-to-Pod Traffic for Some Nodes with more than one ips with different CIDR notation](#no-pod-to-pod-traffic-for-some-nodes-with-more-than-one-ips-with-different-cidr-notation)
+  - [No Pod-to-Pod Traffic for Some Nodes with More Than One Network Interface](#no-pod-to-pod-traffic-for-some-nodes-with-more-than-one-network-interface)
+  - [No Pod-to-Pod Traffic for Some Nodes with More Than One IPs with Different CIDR Notation](#no-pod-to-pod-traffic-for-some-nodes-with-more-than-one-ips-with-different-cidr-notation)
 - [Troubleshooting KubeMarine](#troubleshooting-kubemarine)
   - [Failures During Kubernetes Upgrade Procedure](#failures-during-kubernetes-upgrade-procedure)
-  - [Numerous Generation of Auditd System Messages ](#numerous-generation-of-auditd-system)
+  - [Numerous Generation of Auditd System Messages ](#numerous-generation-of-auditd-system-)
```
Comment (n549) [inline documentation/Troubleshooting.md:31]: this link doesn't work, the trailing hyphen is not needed.

Comment (shmo1218) [reply documentation/Troubleshooting.md]: Removed the hyphen

File: documentation/Troubleshooting.md [hunk 3]
```diff
@@ -551,41 +549,41 @@ Nov 28 14:02:06 node01 kubelet[308309]: E1128 14:02:06.631719  308309 kubelet.go
...
+**Symptoms**: There is no traffic between pods located at different nodes. There is more than 1 IPs on used network interface with different CIDR notations.
 
-**Root cause**: Not all Calico BGP sessions between nodes are established due to different CIDR notations on chosen ips for nodes.
-Typically, such situation can appear in minha scheme with vrrp, where balancer role is combined with other roles. In that case 
-calico can autodetect vrrp for some node instead of its internal ip.
-You can use `calicoctl` to check such situation, e.g. for [example minha cluster.yaml](../examples/cluster.yaml/miniha-cluster.yaml):
+**Root cause**: Not all Calico BGP sessions between nodes are established due to different CIDR notations on the chosen IPs for nodes.
+Typically, such situation can appear in minha scheme with vrrp, where the balancer role is combined with other roles. In that case, 
+Calico can autodetect vrrp for some node instead of its internal IP.
+You can use `calicoctl` to check such a situation. For example, in [example minha cluster.yaml](../examples/cluster.yaml/miniha-cluster.yaml):
```
[the remark this answers is not in the document]
Comment (shmo1218) [reply documentation/Troubleshooting.md]: Corrected the file name

**Review by n549 — APPROVED** [review]

**Pull request merged**